api/OEP_API_tutorial_part3.ipynb

```diff
@@ -75,7 +75,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
```

Comment:
Please clear the output of your notebook before you push it :)

Reply:
:/ I did. Will redo this.